examples/eks/cert-manager.yaml

```diff
@@ -0,0 +1,7221 @@
+# Copyright YEAR The Jetstack cert-manager contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Source: cert-manager/templates/templates.regular.out
```

Comment:
Do we have to copy this large file? Is there any way to refer to it? git submodule maybe?
If it is copied from somewhere else, could it at least be in a third_party directory so that it is easier to update?

Reply:
It's a copy of `config/certmanager/cert-manager.yaml`, it was downloaded from cert-manager official site. We could switch to helm package, but I haven't tried it.